### DISCUSSION TITLE
Can I host the installer tacticalagent-v2.5.0-windows-amd64.exe locally? Or pass a flag for local .exe?

### DISCUSSION THREAD

**Discussion by nries-venio** [amidaware/tacticalrmm#1699] [Q&A]
I have multiple servers that are restricted from outbound 80 and 443. These are intentionally not public facing. I would like to deploy the agent, but it failes when attempting to download the agent from github.

I noticed for meshagent, you can pass a flag to install the meshagent locally, but there doesn't seem to be such a flag for the tacticalagent.

Why does it have to download from github? Why can't the tacticalrmm server host it instead?

ERROR: unable to download agent from https://github.com/amidaware/rmmagent/releases/download/v2.5.0/tacticalagent-v2.5.0
-windows-amd64.exe

**Comment by wh1te909** [comment]
you can host it wherever you want sure. it's planned to have the rmm server host it i'm working on that. in the meantime you can throw the file wherever you want on any web server.
